core-web/libs/edit-content/src/lib/services/dot-edit-content.service.ts

```diff
@@ -185,4 +185,27 @@ export class DotEditContentService {
             })
         );
     }
+
+    /**
+     *
+     *
+     * @return {*}  {Observable<TreeNodeItem>}
+     * @memberof DotEditContentService
+     */
+    getCurrentSite(): Observable<TreeNodeItem> {
```

Comment:
should we call it like `getCurrentSiteAsTreeNodeItem` or something else? `getCurrentSite` is too generic